themes/oceans/catalog.json

```diff
@@ -405,12 +405,6 @@
       "type": "application/json",
       "title": "Global Ocean Heat Content (GOHC)_GRACE/GRACE-FO"
     },
-    {
-      "rel": "child",
-      "href": "../../products/oceanic-current-fronts-world-ocean-circulation/collection.json",
-      "type": "application/json",
-      "title": "Oceanic/current fronts_Gibraltar"
-    },
```

Comment:
I couldn't find this product, only similar ones which did not quite match. Okay to remove?

Reply:
I suppose as well unless @edobrowolska knows more?